### PR TITLE
Update help command and refactor history command

### PR DESCRIPTION
**Pull request by Weiennn**
If everything looks good we can still merge the PR even if codecov fails. 
Reference: https://github.com/nus-cs2103-AY2324S1/forum/issues/248

### CODE REVIEW

**Comment by Chen-Kuei** [comment]
Maybe can also mention the numerical value up to 2 decimal point for Salary attribute.
Another than the above command LGTM!

File: docs/images/helpMessage.png
Comment (Goh-Li-Ting) [inline docs/images/helpMessage.png:1]: Remember to update the image after you update the help command after the other commands have been merged!

File: src/test/java/seedu/address/logic/parser/HistoryCommandParserTest.java [hunk 1]
```diff
@@ -42,5 +46,12 @@ public void parse_invalidArgs_throwsParseException() {
 
         // zero input
         assertParseFailure(parser, "0", String.format(MESSAGE_INVALID_COMMAND_FORMAT, HistoryCommand.MESSAGE_USAGE));
+
+        // greater than max input (Boundary value analysis
```
Comment (Goh-Li-Ting) [inline src/test/java/seedu/address/logic/parser/HistoryCommandParserTest.java:50]: Missing end bracket

Comment (Weiennn) [reply src/test/java/seedu/address/logic/parser/HistoryCommandParserTest.java]: resolved!

**Comment by Weiennn** [comment]
> Maybe can also mention the numerical value up to 2 decimal point for Salary attribute. Another than the above command LGTM!

As in I highlight it in the CommandSummary? I was thinking that it was better to just show how to use it and not the constraints that it has since if I do that then I would have to add the constraints for the rest of the command and this may make CommandSummary cluttered

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM

**Review by ricketytoc — APPROVED** [review]
LGTM
